kclvm/sema/src/advanced_resolver/node.rs

```diff
@@ -867,6 +867,42 @@ impl<'ctx> MutSelfTypedResultWalker<'ctx> for AdvancedResolver<'ctx> {
     }
 
     fn walk_config_expr(&mut self, config_expr: &'ctx ast::ConfigExpr) -> Self::Result {
+        let (start, end) = (self.ctx.start_pos.clone(), self.ctx.end_pos.clone());
```

Comment:
it seems should to impl it in walk_config_entries, rather than walk_config_expr.